addon/mixins/node-actions.js

```diff
@@ -4,19 +4,41 @@ export default Ember.Mixin.create({
     node: null,
     model: null,
     _node: Ember.computed.or('node', 'model'),
-
-    /** Maps all node contributors to format {contribID: Contributor}
-     **/
+    /**
+    * Helper method that maps all node contributors to format {contribId: Contributor}
+    *
+    * @method _generateContributorMap
+    * @param {Object} contributors All contributor relationships on the node
```

Comment:
You're right, I guess contributors are an array.  Every place where it was a resource instead of a string/boolean/date, I just called it an object.  I will go through and see where else I made this mistake.